README.md

```diff
@@ -90,12 +90,16 @@ Configure the server by copying the `lldap_config.docker_template.toml` to
 Environment variables should be prefixed with `LLDAP_` to override the
 configuration.
 
+If the `lldap_config.toml` not existed or created beforehand LLDAP will use default one, with `password` as the default password, after login you can update admin password via web interface.
```

Comment:
"If the `lldap_config.toml` doesn't exist when starting up, LLDAP will use the default one. The default admin password is `password`, you can change it using the web interface.